csrc/scheduler/multi_matmul.cpp

```diff
@@ -1050,7 +1081,154 @@ class MultipleMatmulScheduler {
   // MmaOperand contains only A and B. If tvs are outputs (i.e. not operands),
   // then operand_type should be std::nullopt.
   void scheduleMmaResults() {
-    NVF_THROW("scheduleMmaResults is not yet implemented");
+    auto all_merged_roles = blockTileTensors(mma_results_);
+    for (size_t i : c10::irange(mma_results_.size())) {
+      TensorView*& mma_result = mma_results_[i];
+      std::vector<MatmulDimRole>& merged_roles = all_merged_roles[i];
+
+      // do split-K rFactor to define splitk_sum and smem_epilogue
+      if (params_->splitk_factor != 1) {
+        TensorView* splitk_sum = mma_result->rFactor({-4, -1});
```

Comment:
Who did the split for split-k? Is it `blockTileTensors`? How about adding this information as a comment in code?